MONOREPO.md

```diff
@@ -1,67 +1,28 @@
-# loopback-next
+# Monorepo overview
 
-The [loopback-next](https://github.com/strongloop/loopback-next) repository uses
-[lerna](https://lernajs.io/) to manage multiple packages for LoopBack 4.
+The [loopback-next](https://github.com/strongloop/loopback-next) repository uses [lerna](https://lernajs.io/) to manage multiple packages for LoopBack 4.
 
-## Packages
+<!-- PLEASE KEEP THE TABLE ROWS SORTED ALPHABETICALLY BY PACKAGE NAME-->
```

Comment:
Hmm, I intentionally sorted the packages by the nature of the modules such as  common utilities, core, extensions, examples.